README.md

```diff
@@ -1,3 +1,13 @@
 # emoji-supply
 
 Code for http://emoji.supply/wallpaper
+
+# Dev
+
+```
+python3 -m http.server 8989 --directory docs
```

Comment:
or just open docs/wallpaper/index.html in Chrome directly, that's what I do

Reply:
Oh ... that's so obvious.